docassemble/ALToolbox/data/questions/income.yml

```
@@ -0,0 +1,22 @@
+---
+variable name: al_times_per_year_list
+data:
+  - 
+    - 12
+    - "Monthly"
+  - 
+    - 1
+    - "Yearly"
+  - 
+    - 52
+    - "Weekly"
+  - 
+    - 24
+    - "Twice per month"
+  - 
+    - 26
+    - "Once every two weeks"
+  - 
+    - 4
+    - "Once every 3 months"
```

This should be the same thing in YAML, and it's a lot more concise and a bit less weird (the `-` with nothing else on the line is particularly off IMO)
```suggestion
  - [12, "Monthly"]
  - [1, "Yearly"]
  - [52, "Weekly"]
  - [24, "Twice per month"]
  - [26, "Once every two weeks"]
  - [4, "Once every 3 months"]
```